src/providers/flagsView.ts

```diff
@@ -435,24 +435,24 @@ export class LaunchDarklyTreeViewProvider implements vscode.TreeDataProvider<Fla
 		/**
 		 * Build Fallthrough view
 		 */
-		const fallThrough = envConfig.fallthrough;
-		if (fallThrough.variation !== undefined) {
-			const fallThroughVar = flag.variations[fallThrough.variation];
+		const defaultRule = envConfig.fallthrough;
```

Comment:
envConfig.fallthrough is one example of the many places in the code we use the term 'fallthrough', I tried to change envConfig.fallthrough to envConfig.defaultRule but it seemed like a more difficult change than we wanted in this PR.

Another customer-facing instance of 'Fallthrough' are in the commands